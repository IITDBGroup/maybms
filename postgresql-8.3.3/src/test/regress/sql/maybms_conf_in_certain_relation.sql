--test of error messges of confidence computation on top of certain relation 

create table r(a int, b int);

select conf() from r;

select tconf() from r;

select aconf(.1,.1) from r;

select ecount() from r;

select esum(a) from r;

select a, conf() from r group by a;

select a, tconf() from r;

select a, aconf() from r group by a;

select a, ecount() from r group by a;

select a, esum(b) from r group by a;

drop table r;

create table r(a int, b int);
insert into r values (1,1), (1,2), (2,1), (2,2);

create table s as repair key a in r;

select conf() from( select conf() from s ) as s;

select tconf() from( select tconf() from s ) as s;

select aconf(.1,.1) from( select aconf(.1,.1) from s ) as s;

select ecount() from( select ecount() from s ) as s;

select esum(esum) from( select esum(a) from s ) as s;

drop table r;
drop table s;
