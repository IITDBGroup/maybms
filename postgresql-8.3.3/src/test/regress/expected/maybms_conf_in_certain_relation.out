--test of error messges of confidence computation on top of certain relation 
create table r(a int, b int);
select conf() from r;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
select tconf() from r;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
select aconf(.1,.1) from r;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
select ecount() from r;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
select esum(a) from r;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
select a, conf() from r group by a;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
select a, tconf() from r;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
select a, aconf() from r group by a;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
select a, ecount() from r group by a;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
select a, esum(b) from r group by a;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
drop table r;
create table r(a int, b int);
insert into r values (1,1), (1,2), (2,1), (2,2);
create table s as repair key a in r;
select conf() from( select conf() from s ) as s;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
select tconf() from( select tconf() from s ) as s;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
select aconf(.1,.1) from( select aconf(.1,.1) from s ) as s;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
select ecount() from( select ecount() from s ) as s;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
select esum(esum) from( select esum(a) from s ) as s;
ERROR:  Query not supported: tconf, conf, aconf, esum and ecount cannot used be in a certain query
drop table r;
drop table s;
